test/controllers/HasCorrectGoodsControllerSpec.scala

```diff
@@ -84,10 +84,10 @@ class HasCorrectGoodsControllerSpec extends SpecBase with MockitoSugar {
 
       val commodity   = Commodity("654321", "Description")
       val userAnswers = emptyUserAnswers
-        .set(HasCorrectGoodsPage, true)
```

Comment:
all these tests were failing because now the ORDER you add the userAnswers matter as CommodityCodePage code will always clear HasCorrectGoodsPage